### PR TITLE
Update ru_ru.json

### PR DESCRIPTION
**Pull request by kyrtion**
Fixed grammars language and translated, added new loading tips.

### CODE REVIEW

File: src/main/resources/assets/bedrockify/lang/ru_ru.json [hunk 1]
```diff
@@ -1,40 +1,154 @@
...
+  "bedrockify.loadingTips.89": "Трезубцы выпадают из утопленников. Их можно держать и метать.",
+  "bedrockify.loadingTips.90": "В печке можно расплавить 20 элементов, используя блок сушеных водорослей как топливо.",
+  "bedrockify.loadingTips.91": "Наборы текстур меняют внешний вид блоков, предметов и меню в игре. Попробуйте их!",
+  "bedrockify.loadingTips.92": "Присоединяйтесь к обсуждению на discord.gg/Minecraft",
+  "bedrockify.loadingTips.93": "Изумрудные руды самые редкие в Minecraft! Они встречаются только на высокогорьях!",
+  "bedrockify.loadingTips.94": "Оцелоты задают трепку криперам!",
+  "bedrockify.loadingTips.95": "Диорит – его или любят, или ненавидят.",
+  "bedrockify.loadingTips.96": "Фантомы – это опасные мобы, которые являются по ночам. Не забывайте высыпаться!",
+  "bedrockify.loadingTips.97": "Зомби превращаются в утопленников, когда тонут.",
+  "bedrockify.loadingTips.98": "Зомби превращаются в утопленников, когда тонут.",
```
Comment (juancarloscp52) [inline src/main/resources/assets/bedrockify/lang/ru_ru.json:141]: This one is duplicate of 97 and does not correspond with the EN_US one.

**Comment by juancarloscp52** [comment]
Thanks for taking the time to check and translate all the loading tips! These changes will be included in the next BedrockIfy update.

File: src/main/resources/assets/bedrockify/lang/ru_ru.json (hunk 1 above)
Comment (kyrtion) [reply src/main/resources/assets/bedrockify/lang/ru_ru.json:141]: Oops! ill fix

**Comment by kyrtion** [comment]
Fixed, thank you for created special mod's <3

File: src/main/resources/assets/bedrockify/lang/ru_ru.json (hunk 1 above)
Comment (kyrtion) [reply src/main/resources/assets/bedrockify/lang/ru_ru.json]: fixed

**Pull request merged**